src/AWS.Deploy.Common/Recipes/Validation/RegexValidator.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.\r
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Text.RegularExpressions;
+
+namespace AWS.Deploy.Common.Recipes.Validation
+{
+    public class RegexValidator : IOptionSettingItemValidator
```

Comment:
can you add some docs to this class to explain what the validator is for?

Reply:
Fixed